cassandra/pom.xml

```diff
@@ -76,6 +76,11 @@ LICENSE file.
 	    <type>zip</type>
 	    <scope>test</scope>
 	  </dependency>
+     <dependency>
+       <groupId>com.datastax.cassandra</groupId>
+       <artifactId>cassandra-driver-core</artifactId>
+       <version>3.3.0</version>
+     </dependency>
```

Comment:
This dependency is already here at the top of these dependencies. If you need `3.3.0`, then find and change `cassandra.cql.version`